apps/meeting/scripts/setup_chime_react_submodule.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const { spawnSync } = require('child_process');
+const path = require('path');
+
+function spawnOrFail(command, args, options, printOutput = true) {
+  options = {
+    ...options,
+    shell: true
+  };
+  const cmd = spawnSync(command, args, options);
+
+  if (cmd.error) {
+    console.log(`Command ${command} failed with ${cmd.error.code}`);
+    process.exit(255);
+  }
+  const output = cmd.output.toString();
+  if (printOutput) {
+    console.log(output);
+  }
+  if (cmd.status !== 0) {
+    console.log(`Command ${command} failed with exit code ${cmd.status} signal ${cmd.signal}`);
+    console.log(cmd.stderr.toString());
+    process.exit(cmd.status);
+  }
+  return output;
+}
+
+process.chdir(path.join(__dirname, '../../../amazon-chime-sdk-component-library-react'));
+
+if (!fs.existsSync('node_modules')) {
```

Comment:
Should we also handle, if the node_modules exists and there are changes in package.json then we need to re-install?

Reply:
May be do this everytime? Or have to add details to do so if any changes in react component library.

Reply:
Why do we need to reinstall? This is just for the submodule.

Reply:
So you are changing into the `amazon-chime-component-library..` and then you are checking for `node_module` if not existing and then only run the rest commands, hence, what would be the case:
1. To update the component library to a newer commit?
2. If the newer commit contains a new dependency in component library and locally I have the `node_modules` already it wont update right?

Reply:
You have to manually do it in that case anyway. This is purely a setup step to run it. I think it is the same way in JS SDK demo (we don't run npm install every time).

Reply:
Ok, yeah I think my concern is just to let builders know how to update it :)